docs/content/guides/monitoring-and-tracing.md

```diff
@@ -148,68 +140,3 @@ If configured correctly, tracing data that is generated or propagated by the NGI
 
 {{< img src="/img/opentelemetry.png" alt="OpenTelemetry Data Flow" >}}
 *Tracing data flow using the OpenTelemetry Collector*
```

Comment:
We still want this image, this is for otel

Reply:
its still there